Add HTML, CSS and media files

rpire opened the pull request:
### Files added:

- Added index.html file.
- Added style.css file.
- Added "Media" folder with the required images.

> Note: the HEX color code #C4C4C4 was used for the menu element even if it didn't match the color shown in the Figma template.

rpire commented:
the Linter check has failed without giving me any error message. I will close the pull and push again.

rpire commented:
## Issue

I had a problem with the checks, but after closing and re-opening the PR the code passed all the Linter checks.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,33 @@
+      <div id="burger-bar-2" class="burger-element"></div>
+      <div id="burger-bar-3" class="burger-element"></div>
+    </div>
+    <h1>Charlie Giorgania</h1>
+    <p>
+      Hello! I am a software developer! I can help you build a product, feature
+      or website. Taka a look of my works. If you like what you see and have a
+      project you need coded, don’t hesitate and contact me.
+    </p>
```
- [ ] This section should be enclosed inside` <section> </section>`.

rpire replied:
Change request satisfied.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,33 @@
+      <a id="collab-link" href="#" rel="noopener noreferrer">
+        <div id="collab-button">Start Collaboration</div>
+      </a>
+      <img id="mosaic-img" src="/Media/image_geometry_2.svg" alt=" ">
```
- [ ] always add` ./` before the path to let the browser know the exact location of the file.

rpire replied:
Change request satisfied.

arslanbisharat commented on style.css:
```diff
@@ -0,0 +1,97 @@
+#burger-container {
```
- [ ] Please don't use id as a selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

rpire replied:
change request satisfied.

arslanbisharat commented on style.css:
```diff
@@ -0,0 +1,97 @@
+  align-items: flex-end;
+}
+
+#burger-bar-2 {
```
- [ ] Please don't use id as selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

rpire replied:
change request satisfied.

arslanbisharat commented on style.css:
```diff
@@ -0,0 +1,97 @@
+  margin-top: 4px;
+}
+
+#burger-bar-3 {
```
- [ ] Please don't use id as selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

rpire replied:
Change request satisfied.

arslanbisharat commented on style.css:
```diff
@@ -0,0 +1,97 @@
+  margin-bottom: 2px;
+}
+
+#collaboration {
```
- [ ] Please don't use id as selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

rpire replied:
Change request satisfied.

arslanbisharat commented on style.css:
```diff
@@ -0,0 +1,97 @@
+#collab-link {
+  width: 90%;
+  height: 50px;
+  margin: 10% 5%;
```
- [ ] Please don't use id as selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

rpire replied:
Change request satisfied.

arslanbisharat commented on style.css:
```diff
@@ -0,0 +1,97 @@
+  margin: 10% 5%;
+}
+
+#collab-button {
```
- [ ] Please don't use id as selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

rpire replied:
Change request satisfied.

arslanbisharat commented on style.css:
```diff
@@ -0,0 +1,97 @@
+  height: 100%;
+}
+
+#collab-button:hover {
```
- [ ] Please don't use id as selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

rpire replied:
Change request satisfied.

arslanbisharat commented on style.css:
```diff
@@ -0,0 +1,97 @@
+  border: 2px solid #1a2236;
+}
+
+#mosaic-img {
```
- [ ] Please don't use id as selector. if you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS.

rpire replied:
Change request satisfied.

arslanbisharat reviewed (changes requested):
# Changes Requested , 

Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
 
- [ ] Please follow the given designs for the project. I am sorry to inform you but I couldn't find which design you choose from [here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 
 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

rpire commented:
## Hello @arslanbisharat !

Thank you for reviewing my project so thoroughly. I noticed that you couldn't find the template from which I'm building my portfolio. It is one of the five Figma templates from which we could choose for this project, I will be leaving the link over [here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=67%3A1095).

I will proceed to make the requested changes, improve the semantics of my webpage and re-submit my project.

Thanks again, happy coding! 🤓🚀

rpire commented:
## Commit [b9364e3](https://github.com/rpire/microverse-portfolio/pull/1/commits/b9364e3a91172c363e06fd67656774cc6ca6503e) changes:

- Enclosed the `h1` and `p` elements between `section` tags.
- Added ./ before every path.
- Changed the use of ID selectors for "class" selectors.

## Commit [32bf40d](https://github.com/rpire/microverse-portfolio/pull/1/commits/32bf40d803f5b6ef5427da46e7c7b191baa70259) changes: 

- Removed a space in line 40 causing a Linter error.

Emmyn5600 reviewed (approved):
## STATUS: Approved :clap: :clap:

### Hi @rpire :raised_hands:

Congratulations!:tada:
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Great work on implementing the changes requested by the previous code reviewer.

###  Optional but I highly recommend you to work on it before you merge it.

- [ ] Work on your Readme file, for example your title is to review the project from trial instead of being Personal Portfolio or any descriptive title.
![readmefile](https://user-images.githubusercontent.com/55635977/124912936-d42f1d00-dfee-11eb-8bda-76f2a046cdce.png)
- [ ] Remove the underline under start collaboration button
![remove](https://user-images.githubusercontent.com/55635977/124913324-49025700-dfef-11eb-9870-d6b242965804.png)



_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

rpire commented:
## Hi @Emmyn5600 🎉

Thank you for reviewing my project!

I will correct the title for this project since I copied the title of the project before this and forgot to modify it, thank you for noticing.

I will also remove the underline from the `a` element you mention. I will make sure to implement these changes before merging.

Once more thank you, and until next time! 👍

Emmyn5600 commented:
Thank you too @rpire, all the best.

rpire commented:
## Commit [6c87fc3](https://github.com/rpire/microverse-portfolio/pull/1/commits/6c87fc39167fa97264732d314642a43cd95948f9) changes:

- Modified title in README.md file.
- Removed the underline from the button's text.
